defaults/api_ents_test.go

```diff
@@ -0,0 +1,31 @@
+package defaults
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/docker/libentitlement/secprofile"
+	"github.com/docker/libentitlement/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestApiEntitlementEnforce(t *testing.T) {
+	entitlementID := APIEntAllowID
+	require.NotNil(t, DefaultEntitlements[entitlementID])
+
+	ociProfile := secprofile.NewOCIProfile(testutils.TestSpec(), "test-profile")
+	apiEnt := DefaultEntitlements[entitlementID]
+
+	_, err := apiEnt.Enforce(ociProfile)
+	require.Equal(t, fmt.Errorf("OCI profile's APIAccess field nil"), err)
```

Comment:
How do I set up the ociprofile so its `APIAccess` field is not nil?

Reply:
We didn't finish this part last time, it was in the todo list iirc. `NewOCIProfile()` would need an additional parameter.